### ISSUE TITLE
REST API Entries Event 500 ERROR

### ISSUE THREAD

**Issue by zooshme** [symphonists/rest_api#11]
I've just installed the REST API Extension in Symphony 2.3.2 but I get 500 Error when accessing the REST API Entries Event. How could I make the browser give me a full error report rather than a blank page in Chrome? Also any ideas what the error could be and how to fix it?

Many thanks.

**Comment by manuelro** [comment]
The same here. All I get is a blank page.

**Comment by nitriques** [comment]
Do you have some logs ?

**Comment by manuelro** [comment]
Mine says:

```
[Fri Aug 07 14:52:42.083051 2015] [:error] [pid 1133] [client 127.0.0.1:58423] PHP Fatal error:  Class 'Symphony' not found in /www/sites/sym/wwwroot/symphony/lib/boot/bundle.php on line 18
```

**Comment by nitriques** [comment]
Can you replace those 3 lines https://github.com/symphonists/rest_api/blob/master/handler.php#L4-L6 with those https://github.com/symphonycms/symphony-2/blob/master/index.php#L6-L10

**Comment by manuelro** [comment]
Did just what you said, the error is gone but I got another one:

```
Symphony Fatal Error: No suitable engine object found

An error occurred in /www/sites/sym/wwwroot/symphony/lib/core/class.symphony.php around line 147

142 if (class_exists('Administration', false)) {
143 return Administration::instance();
144 } elseif (class_exists('Frontend', false)) {
145 return Frontend::instance();
146 } else {
147 throw new Exception(__('No suitable engine object found'));
148 }
149 }
150
151 /**
Backtrace
[/www/sites/sym/wwwroot/extensions/rest_api/lib/class.rest_api.php:17]
    Symphony::Engine();
[/www/sites/sym/wwwroot/extensions/rest_api/lib/class.rest_api.php:54]
    REST_API::__authenticate();
[/www/sites/sym/wwwroot/extensions/rest_api/handler.php:12]
    REST_API::init();
Database Query Log
[0.0003] SET character_set_connection = 'utf8', character_set_database = 'utf8', character_set_server = 'utf8';
[0.0001] SET CHARACTER SET 'utf8';
[0.0001] SET time_zone = '+00:00';
[0.0015] SELECT SQL_CACHE t1.name, t2.page, t2.delegate, t2.callback FROM `sym_extensions` as t1 INNER JOIN `sym_extensions_delegates` as t2 ON t1.id = t2.extension_id WHERE t1.status = 'enabled' ORDER BY t2.delegate, t1.name;
```

**Comment by nitriques** [comment]
Could you try to add this line ? 
https://github.com/symphonists/rest_api/blob/master/handler.php#L5 (uncommented)

**Comment by nitriques** [comment]
@manuelro ping!

**Comment by nitriques** [comment]
Fixed in 9319dbdd51
